corehq/apps/export/forms.py

```diff
@@ -919,30 +921,38 @@ def get_filters(self, can_access_all_locations, accessible_location_ids, group_i
         :param location_ids:
         :param date_range: A DatePeriod or DateSpan
         """
-        form_filters = []
-        if can_access_all_locations:
-            form_filters += [_f for _f in [
-                self._get_group_filter(group_ids),
-                self._get_user_type_filter(user_types),
-            ] if _f]
-
-        form_filters += [_f for _f in [
-            self._get_users_filter(user_ids),
-            self._get_locations_filter(location_ids)
-        ] if _f]
-
-        form_filters = flatten_non_iterable_list(form_filters)
-        if form_filters:
-            form_filters = [OR(*form_filters)]
-        else:
-            form_filters = []
+
+        form_filters = filter(None, [self._create_user_filter(user_types, user_ids, group_ids, location_ids)])
```

Comment:
This will need to change to `list(filter(...))` or  a list comprehension for python 3. `filter` in py3 is lazy evaluation by default so it doesn't return a list